app/upgrades/v15/upgrades.go

```diff
@@ -18,6 +20,113 @@ import (
 	poolmanagertypes "github.com/osmosis-labs/osmosis/v14/x/poolmanager/types"
 )
 
+// an array of 10 strings containing the numbers from 1 to 10:
+
+// Using the same queries as stargate_whitelist.go
+var whitelistedQueries = []string{
+	// cosmos-sdk queries
+
```

Comment:
Im wondering if there is a way to use the querypath in stargate whitelist directly? (since the stargatewhitelist is a map, would it be possible to directly load the keys of the map?)

Reply:
yeah, I wanted to to make it dynamic first, but then got concerned that there might be issues later if the Map changes (and someone wants to recreate the chain by running the upgrade)